examples/nextjs/pages/index.tsx

```diff
@@ -1,27 +1,8 @@
-import { useActiveProfile, useWalletLogin } from '@lens-protocol/react';
 import Head from 'next/head';
-import { useAccount, useConnect } from 'wagmi';
-import { InjectedConnector } from 'wagmi/connectors/injected';
 
-export default function Home() {
-  const { execute: login, isPending } = useWalletLogin();
-  const { data: profile } = useActiveProfile();
-
-  const { isDisconnected } = useAccount();
-  const { connectAsync } = useConnect({
-    connector: new InjectedConnector(),
-  });
-  const onLoginClick = async () => {
-    if (isDisconnected) {
-      const { connector } = await connectAsync();
-
-      if (connector instanceof InjectedConnector) {
```

Comment:
I updated the login flow implementation in nextjs app to use the same components as in web-wagmi, refactored also a bit component structure in web-wagmi